using-timescaledb/distributed-hypertables.md

```diff
@@ -103,6 +103,90 @@ the data node's data is outside the retention window,
 [`drop_chunks`][drop_chunks] can be used to delete the data on the data
 node.
 
+### Replication Factor for Distributed Hypertables [](replication-factor)
```

Comment:
IMO, a section like this should be about native replication in general and not the "replication factor" specifically.

Reply:
Changed header.